package.json

```diff
@@ -3,7 +3,12 @@
   "version": "0.0.0-semantic-releasing",
   "description": "Annotate UI components and link them to wherever you want.",
   "scripts": {
```

Comment:
Change `"build:dist"` to `"prebuild": "mkdir dist/"` and maybe remove `dist/` first to ensure you always get a clean build?